client/src/app/pages/applications/analysis-wizard/custom-rules.tsx

```diff
@@ -195,6 +195,22 @@ export const CustomRules: React.FC<CustomRulesProps> = (props) => {
                 type="button"
                 variant="plain"
                 onClick={() => {
+                  customRulesFiles.forEach((file) => {
+                    const { source, target } = parseRules(file);
+                    if (source && formSources.includes(source)) {
+                      const updatedSources = formSources.filter(
+                        (source) => source !== source
```

Comment:
Need to change the iterator var so one can understand what is what.

Reply:
Good catch.